### PR TITLE
Documentation for new Card Fields Component

### PR DESCRIPTION
**Pull request by jplukarski**
### Description

Initial docs for capturing a payment with the new Card Fields component. We used the [spec repo](https://github.com/paypal/paypal-sdk-spec/tree/main/spec/client/javascript) to guide us in making these docs

### Why are we making these changes? Include references to any related Jira tasks or GitHub Issues

[DTNOR-748](https://paypal.atlassian.net/browse/DTNOR-748)

@paypal/js-sdk-maintainers 

❤️  Thank you!

### CODE REVIEW

File: src/card/docs/Initialization/README.md [hunk 1]
```diff
@@ -0,0 +1,298 @@
+# JS SDK Initialization
+
+## Script Tag
+
+* Load the SDK from `paypal.com/sdk/js?client-id=xyz`
+* This provides the `window.paypal` namespace
+
+
+```html
+<script src="https://www.paypal.com/sdk/js?client-id=<YOUR_CLIENT_ID>" />
```
Comment (wsbrunson) [inline src/card/docs/Initialization/README.md:10]:
```suggestion
<script src="https://www.paypal.com/sdk/js?client-id=<YOUR_CLIENT_ID>&components=card-fields" />
```

File: src/card/docs/Initialization/README.md [hunk 2]
```diff
@@ -0,0 +1,298 @@
...
+
+## Initialization Parameters
+
+Initialization Parameters can be passed via the script url query or via data attributes
+
+```html
+<script
+    src="https://www.paypal.com/sdk/js?client-id=test&components=card-fields"
+    data-user-id-token="xyz"></script>
+```
```
Comment (wsbrunson) [inline src/card/docs/Initialization/README.md:25]: you say the same thing right below

File: src/card/docs/Initialization/README.md [hunk 3]
```diff
@@ -0,0 +1,298 @@
...
+Pass in a custom country to inform the SDK of the geolocation of the user. Used primarily for eligibility.
+
+Available countries: See [Country Support](./support/countries.md)
+
+Example:
+
+```html
+<script src="https://www.paypal.com/sdk/js?client-id=test&buyer-country=FR"></script>
+```
+
```
Comment (wsbrunson) [inline src/card/docs/Initialization/README.md:99]: I would omit this, this query parameter only works in the Sandbox environment

**Review by wsbrunson — COMMENTED** [review]
other than these comments looks great!

**Review by remotevision — APPROVED** [review]
👍

**Review by wsbrunson — APPROVED** [review]

**Pull request merged**